### PR TITLE
[TRANSLATION] Polish translation update

### PR DESCRIPTION
**Pull request by archeYR**
Polish translation update for drwtsn32, winlogon and usersrv.

### CODE REVIEW

File: base/applications/drwtsn32/lang/pl-PL.rc [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/*
+ * PROJECT:     Dr. Watson crash reporter
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     en-US resource file
+ * COPYRIGHT:   Copyright 2018 William Kent (wjk011 [at] gmail [dot] com)
```
Comment (learn-more) [inline base/applications/drwtsn32/lang/pl-PL.rc]: I don't think you are William kent?

**Comment by HBelusca** [comment]
Hi, we get compile errors:
```
c:\reactos-cov\base\applications\drwtsn32\lang/pl-PL.rc(9) : error RC2151 : cannot reuse string constants, 101(0x65) - "ReactOS Crash Reporter" already defined. Cannot add 101(0x65) - "Raportowanie b┼
c:\reactos-cov\base\applications\drwtsn32\lang/pl-PL.rc(10) : error RC2151 : cannot reuse string constants, 102(0x66) - "The application %ls has crashed. Information about this crash has been saved to:
```

File: win32ss/user/winsrv/usersrv/lang/pl-PL.rc [hunk 2]
```diff
@@ -34,16 +34,16 @@ END
 
 STRINGTABLE
 BEGIN
-    IDS_SEVERITY_SUCCESS        "Success"
-    IDS_SEVERITY_INFORMATIONAL  "System Information"
-    IDS_SEVERITY_WARNING        "System Warning"
-    IDS_SEVERITY_ERROR          "System Error"
+    IDS_SEVERITY_SUCCESS        "Skukces"
```
Comment (HBelusca) [inline win32ss/user/winsrv/usersrv/lang/pl-PL.rc]: Typo.

File: win32ss/user/winsrv/usersrv/lang/pl-PL.rc [hunk 3]
```diff
@@ -34,16 +34,16 @@ END
...
+    IDS_SEVERITY_ERROR          "Błąd"
 END
 
 STRINGTABLE
 BEGIN
-    IDS_SYSTEM_PROCESS          "System Process"
-    IDS_UNKNOWN_SOFTWARE_EXCEPT "unknown software exception"
-    IDS_OK_TERMINATE_PROGRAM    "Click on OK to terminate the program."
-    IDS_CANCEL_DEBUG_PROGRAM    "Click on CANCEL to debug the program."
+    IDS_SYSTEM_PROCESS          "Proces"
```
Comment (HBelusca) [inline win32ss/user/winsrv/usersrv/lang/pl-PL.rc]: How do you convey the idea that this process is not any process but the "System" (aka. kernel-mode) process?
Note that the task manager translates this as "Proces bezczynności systemu" (for "System Idle Process").

File: win32ss/user/winsrv/usersrv/lang/pl-PL.rc [hunk 4]
```diff
@@ -34,16 +34,16 @@ END
...
+    IDS_UNKNOWN_SOFTWARE_EXCEPT "nieznany wyjątek programowy"
+    IDS_OK_TERMINATE_PROGRAM    "Kliknij OK aby zakończyć pracę programu."
+    IDS_CANCEL_DEBUG_PROGRAM    "Kliknij Anuluj aby uzyskać więcej informacji o błędzie."
```
Comment (HBelusca) [inline win32ss/user/winsrv/usersrv/lang/pl-PL.rc]: Is "Anuluj" the very same word as the one used in the "Cancel" buttons of message boxes in Polish?

File: base/applications/drwtsn32/drwtsn32.rc [hunk 5]
```diff
@@ -19,3 +19,6 @@
 #ifdef LANGUAGE_EN_US
 	#include "lang/en-US.rc"
 #endif
+#ifdef LANGUAGE_PL_PL
+	#include "lang/pl-PL.rc"
+#endif
```
Comment (HBelusca) [inline base/applications/drwtsn32/drwtsn32.rc]: Can you fix also the indentation here and use 4 spaces instead? (please fix also in the english language case :) )

File: base/applications/drwtsn32/lang/pl-PL.rc [hunk 6]
```diff
@@ -0,0 +1,11 @@
+/*
+ * PROJECT:     Dr. Watson crash reporter
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     pl-PL resource file
+ */
+
```
Comment (HBelusca) [inline base/applications/drwtsn32/lang/pl-PL.rc]: As in all language files, your file misses the obligatory line:
```
LANGUAGE LANG_POLISH, SUBLANG_DEFAULT
```
(and I know, the english language file misses its own too:
```
LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
```
)

File: base/applications/drwtsn32/drwtsn32.rc (hunk 5 above)
Comment (archeYR) [reply base/applications/drwtsn32/drwtsn32.rc]: Sure :)

File: win32ss/user/winsrv/usersrv/lang/pl-PL.rc (hunk 4 above)
Comment (archeYR) [reply win32ss/user/winsrv/usersrv/lang/pl-PL.rc]: Yes.

File: win32ss/user/winsrv/usersrv/lang/pl-PL.rc (hunk 3 above)
Comment (archeYR) [reply win32ss/user/winsrv/usersrv/lang/pl-PL.rc]: I think it should be "Proces systemowy" then.

**Pull request merged**